src/sentry/templates/sentry/js-sdk-loader.min.js.tmpl

```diff
@@ -1,4 +1,4 @@
-r?r.apply(b,d[c].e):"p"in d[c]&&B&&B.apply(b,[d[c].p])}catch(y){console.error(y)}}var q=n,C=!1;for(n=0;n<document.scripts.length;n++)if(-1<document.scripts[n].src.indexOf(E)){q&&"no"===document.scripts[n].getAttribute("data-lazy")&&(q=!1);break}var A=!1,t=[];p.data=[];b[l]=b[l]||{};b[l].v=function(a){t.push(a);q&&!C||v(t)};b[l].o=function(){C=!0;q&&setTimeout(function(){v(t)})};"init addBreadcrumb captureMessage captureException captureEvent configureScope withScope showReportDialog".split(" ").forEach(function(a){b[l][a]=
-function(){p({f:a,a:arguments})}});var w=b[f];b[f]=function(){p({e:[].slice.call(arguments)});w&&w.apply(b,arguments)};b[f].g=!0;var x=b[g];b[g]=function(a){p({p:"reason"in a?a.reason:"detail"in a&&"reason"in a.detail?a.detail.reason:a});x&&x.apply(b,arguments)};b[g].g=!0;q||setTimeout(function(){v(t)})})(window,document,"script","onerror","onunhandledrejection","Sentry","{{ publicKey|safe }}","{{ jsSdkUrl|safe }}",{{ config|to_json|safe }},{{ isLazy|safe|lower }});
+{% load sentry_helpers %}(function(c,w,C,q,r,k,D,E,x,l){function t(a){if(!y){y=!0;var f=w.scripts[0],d=w.createElement(C);d.src=E;d.crossOrigin="anonymous";d.addEventListener("load",function(){try{c[q]=u;c[r]=v;c.SENTRY_SDK_SOURCE="loader";var b=c[k],g=b.init;b.init=function(e){for(var h in e)Object.prototype.hasOwnProperty.call(e,h)&&(x[h]=e[h]);g(x)};F(a,b)}catch(e){console.error(e)}});f.parentNode.insertBefore(d,f)}}function F(a,f){try{for(var d=m.data,b=0;b<a.length;b++)if("function"===typeof a[b])a[b]();
+var g=!1,e=c.__SENTRY__;"undefined"!==typeof e&&e.hub&&e.hub.getClient()&&(g=!0);e=!1;for(b=0;b<d.length;b++)if(d[b].f){e=!0;var h=d[b];!1===g&&"init"!==h.f&&f.init();g=!0;f[h.f].apply(f,h.a)}!1===g&&!1===e&&f.init();var z=c[q],A=c[r];for(b=0;b<d.length;b++)"e"in d[b]&&z?z.apply(c,d[b].e):"p"in d[b]&&A&&A.apply(c,[d[b].p])}catch(G){console.error(G)}}var n=l,B=!1;for(l=0;l<document.scripts.length;l++)if(-1<document.scripts[l].src.indexOf(D)){n&&"no"===document.scripts[l].getAttribute("data-lazy")&&
+(n=!1);break}var y=!1,p=[],m=function(a){("e"in a||"p"in a||a.f&&-1<a.f.indexOf("capture")||a.f&&-1<a.f.indexOf("showReportDialog"))&&n&&t(p);m.data.push(a)};m.data=[];c[k]=c[k]||{};c[k].onLoad=function(a){p.push(a);n&&!B||t(p)};c[k].forceLoad=function(){B=!0;n&&setTimeout(function(){t(p)})};"init addBreadcrumb captureMessage captureException captureEvent configureScope withScope showReportDialog".split(" ").forEach(function(a){c[k][a]=function(){m({f:a,a:arguments})}});var u=c[q];c[q]=function(a,
```

Comment:
The issue here is that `onLoad` stopped being defined, it got minified for some reason.